src/providers/push-notifications/push-notifications.ts

```diff
@@ -224,4 +232,28 @@ export class PushNotificationsProvider {
     if (!this._token) return;
     this.FCMPlugin.clearAllNotifications();
   }
+
+  private verifySignature(data): boolean {
+    const b = BWC.Bitcore;
+    const ECDSA = b.crypto.ECDSA;
+    const Hash = b.crypto.Hash;
+    const SEP = '::';
+    const pubKey =
+      '02463827f64d47a9adaeddbeb860c1ea83eeb19024f9056de1772604c8085340a5';
```

Comment:
please move this to https://github.com/bitpay/copay/blob/master/app-template/bitpay/appConfig.json
marketingPublicKey